src/components/siteContent/SiteContent.js

```diff
@@ -58,8 +47,13 @@ const SiteContent = ({ group1, group2, group3 }) => {
                 onMouseOver={onPhotoHover}
                 onMouseLeave={onMouseLeave}
               />
-              {isDescriptionOpened && index === Number(isPhotoHovered) && (
-                <div className={styles.description}>{isPhotoHovered}</div>
+              {isDescriptionOpened && index === Number(hoveredPhoto) && (
+                <div className={styles.description}>
+                  <p style={{ fontWeight: "bold" }}>
+                    {groups[hoveredPhoto].description}
```

Comment:
hoveredPhotoIndex nie będzie lepiej? hoveredPhoto sugeruje mi, że to zdjęcie całe